File: exercicios1_c/9_ternario.c
#include <stdio.h>  // Inclui a biblioteca de entrada e saída padrão

int main(void) {  // Função principal
  int numero, x;  // Declaração de variáveis

  scanf("%d", &numero);  // Lê um número inteiro

  x = (numero > 10) ? 1 : 0;  // Verifica se o número é maior que 10

  printf("%d", x);  // Exibe o resultado

  return 0;  // Retorna 0 para indicar sucesso
}
/*
input: 12
retorno: 1
*/ 

File: exercicios1_rust/9_ternario.rs
use std::io;

fn main() {
    let mut input = String::new();

    io::stdin().read_line(&mut input).expect("Falha ao ler a entrada");

    let numero: i32 = input.trim().parse().expect("Falha ao converter para número");

    let x = if numero > 10 { 1 } else { 0 };

    println!("{}", x);
}

/*
input: 12
retorno: 1
*/ 